back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -49,51 +49,47 @@
       parameter :review_state, 'Filter by project review status (pending, approved)', required: false
 
       example_request 'List all admin publications' do
+        hidden_project = create(:project, internal_role: 'community_monitor', admin_publication_attributes: { publication_status: 'hidden' })
         expect(status).to eq(200)
-        json_response = json_parse(response_body)
-        expect(json_response[:data].size).to eq 10
-        expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :type) }.count('project')).to eq 8
-        expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :type) }.count('folder')).to eq 2
+        expect(response_data.size).to eq 10
+        expect(response_data.map { |d| d.dig(:relationships, :publication, :data, :type) }.count('project')).to eq 8
+        expect(response_data.map { |d| d.dig(:relationships, :publication, :data, :type) }.count('folder')).to eq 2
+        expect(response_data.pluck(:id)).not_to include(hidden_project.admin_publication.id)
```

Comment:
This is the only actual test change. The rest of the changes are only changes to use the `response_data` helper